sysdump/sysdump.go

```diff
@@ -202,28 +202,35 @@ func NewCollector(k KubernetesClient, o Options, startTime time.Time, cliVersion
 	if c.Options.CiliumOperatorNamespace == "" {
 		ns, err := detectCiliumOperatorNamespace(k)
 		if err == nil {
-			c.log("🔮 Detected Cilium operator in namespace %q", ns)
+			c.log("🔮 Detected Cilium operator in namespace: %q", ns)
 			c.Options.CiliumOperatorNamespace = ns
 		} else {
-			c.log("ℹ️ Failed to detect Cilium  operator ")
+			c.log("ℹ️ Failed to detect Cilium operator")
 		}
 	} else {
 		c.log("ℹ️  Cilium operator namespace: %s", c.Options.CiliumOperatorNamespace)
 	}
 
+	if c.Options.CiliumHelmReleaseName == "" {
+		c.log("ℹ️ Using default Cilium Helm release name: %q", defaults.HelmReleaseName)
+		c.Options.CiliumHelmReleaseName = defaults.HelmReleaseName
```

Comment:
Given that this seems to be the only remaining use of `defaults.HelmReleaseName`, could we make it a private constant in the sysdump package?

Reply:
Localizing this value would reduce potential ambiguity with the cmd.PersistentFlags default value implemented in this PR.